metadeploy/api/views.py

```diff
@@ -207,6 +208,20 @@ def preflight(self, request, pk=None):
         if request.method == "POST":
             return self.preflight_post(request)
 
+    @action(detail=True, methods=["post"])
+    def create_scratch_org(self, request, pk=None):
+        plan = self.get_object()
+        serializer = CreateScratchOrgSerializer(data=request.data)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+        email = serializer.validated_data["email"]
+        org_name = plan.org_name
+        job = create_scratch_org_job.delay(
+            plan_id=str(plan.id), email=email, org_name=org_name
+        )
+        return Response({"job_id": job.id}, status=status.HTTP_202_ACCEPTED)
+
 
 class OrgViewSet(viewsets.ViewSet):
```

Comment:
@wlonk I don't fully understand how the UUID authentication is going to work, but I think we'll want to update this `OrgViewSet` to also include info about a currently-running job on a MD-created scratch org, if the user passes in the UUID token? We'll want some way to know that an org is being created, or a job is being run on a MD-created org.